lambdas/handlers/document_reference_search_handler.py

```diff
@@ -1,2 +1,41 @@
+import json
+
+import boto3
+from boto3.dynamodb.conditions import Key
+import logging
+from botocore.exceptions import ClientError
+
+from enums.metadata_field_names import DynamoField
+from services.dynamo_query_service import DynamoQueryService
+from utils.exceptions import InvalidResourceIdException
+from utils.lambda_response import ApiGatewayResponse
+from utils.nhs_number_validator import validate_id
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Constants
+TABLE_NAME = "dev_DocumentReferenceMetadata"
```

Comment:
Is this hardcoded only for the dev environment? 
What happens in a sandbox / pre-prod environment where the table name is likely "pre-prod_Doc...."

Reply:
@AlexHerbertNHS We can set the table name as environment variable in the lambda with terraform. And you can get it by using 'os.environ[key]'.

Reply:
Done